examples/transformers_custom_handler/README.md

```diff
@@ -0,0 +1,38 @@
+# Text Classfication with huggingface/transformers on the CoLA task using a custom model handler
+
+This is an example to finetune and serve a BERT model from huggingface/transformers 
+for the CoLA task (GLUE). 
+
+The code for finetuning is based on the transformers GLUE CoLA example:
+https://github.com/huggingface/transformers/tree/v2.8.0/examples#glue
+
+Code for a simple custom handler for the resulting BERT model can be found in the 
+file `transformers_cola_handler.py`.
+
+
+# Finetuning
+
```

Comment:
@MFreidank Please add all the required dependencies to the readme prerequisite install steps -- transformers and any others

Reply:
Resolved, added an `Install Dependencies` to my README that installs recommended versions of the libraries.